assets/javascripts/src/components/InputField.jsx

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 
 export default class InputField extends React.Component {
     render() {
-        const { outerClassName, children, ...props } = this.props;
+        const { outerClassName, children, optional, ...props } = this.props;
 
         return <div className={outerClassName || ''}>
-            <label htmlFor={this.props.id} className="label">{this.props.label}</label>
+            <label htmlFor={this.props.id} className="label">{this.props.label}
+                {optional && <span className="label-optional">(optional)</span>}
```

Comment:
There's no need to pass `optional` as an extra prop, you can check against the standard `required` attribute which is passed to `InputField` as `this.props.required`